### PR TITLE
Remove Prometheus

### PR DESCRIPTION
**Pull request by aroltsch**
Remove Prometheus. DD is already hooked up.

Tested in UAT

### CODE REVIEW

File: app/services/vacols_service.rb [hunk 1]
```diff
@@ -72,25 +69,13 @@ def query_service
         order by 2 desc
       SQL
       @wait_time_by_class = @connection.exec_query(query)
-      @wait_time_by_class.each do |wtc|
```
Comment (enriquemanuel) [inline app/services/vacols_service.rb]: are we pushing this metrics to DD?

File: app/services/vacols_service.rb [hunk 2]
```diff
@@ -72,25 +69,13 @@ def query_service
...
-        latency_gauge.set({
```
Comment (enriquemanuel) [inline app/services/vacols_service.rb]: same here, is this going to DD already?

File: app/services/vacols_service.rb [hunk 3]
```diff
@@ -72,25 +69,13 @@ def query_service
...
-      end
-    
+
       # Overall system time that includes DB Time, DB CPU and various metrics
       query = <<-SQL
         select stat_name, value "time"
         from v$sys_time_model
       SQL
       @sys_time_model = @connection.exec_query(query)
-      @sys_time_model.each do |stm|
```
Comment (enriquemanuel) [inline app/services/vacols_service.rb]: another in DD?

File: app/services/vacols_service.rb [hunk 4]
```diff
@@ -101,11 +86,7 @@ def query_service
         order by count(*) desc
       SQL
       @sum_all_db_time_24hrs = @connection.exec_query(query)
-      latency_gauge.set({
```
Comment (enriquemanuel) [inline app/services/vacols_service.rb]: same?

Comment (aroltsch) [reply app/services/vacols_service.rb]: yes, all this metrics is already being pushed to the DD https://github.com/department-of-veterans-affairs/caseflow-monitor/blob/9a2e01e7c0cd3a966ff0a1f709ddeae03d075e64/app/services/vacols_service.rb#L119

**Pull request merged**